docker-run-automated-analysis.sh

```diff
@@ -35,7 +35,7 @@ INPUT_GOOGLE_CLOUD_CREDENTIALS=$2
 INPUT_PIPELINE_CONFIGURATION=$3
 INPUT_MESSAGES_TRACED_DATA=$4
 INPUT_INDIVIDUALS_TRACED_DATA=$5
-OUTPUT_DIR=$6
+AUTOMATED_ANALYSIS_OUTPUT_DIR=$6
```

Comment:
Can you rename <output-dir> on L28 to <automated-analysis-output-dir>

Reply:
updated thanks